### PR TITLE
Updates to R docs

### PR DESCRIPTION
**Pull request by cpsievert**
Just after I made all these changes, I remembered [this consequence of using htmlwidgets](https://github.com/yihui/knitr-jekyll/issues/8#issuecomment-104112826). So, I’m not sure how exactly md -> html happens,  but most likely it's gonna require a messy hack to embed these graphs client-side. :(

### CODE REVIEW

**Comment by cpsievert** [comment]
cc @chriddyp

**Comment by chriddyp** [comment]
@cpsievert - can we still keep an option for printing the object with `plotly_POST

File: _posts/r/2015-07-30-embedding-in-html.Rmd [hunk 1]
```diff
@@ -21,25 +19,21 @@ Sys.setenv("plotly_username"="RPlotBot")
...
-```
+You can also post your figure to a plotly server with `plotly_POST()`. This function returns an object of class figure. When you print a figure object in an rmarkdown/knitr document, it embeds the figure as an iframe (pointing to your figure).
 
-```html
-<img src="https://plot.ly/~RPlotBot/501.png">
+```{r}
+f <- plotly_POST(p, filename = "r-docs/midwest-boxplots")
+class(f)
+f
 ```
```
Comment (chriddyp) [inline _posts/r/2015-07-30-embedding-in-html.Rmd]: hey @cpsievert - this doesn't seem to work for me: 
image

Comment (cpsievert) [inline _posts/r/2015-07-30-embedding-in-html.Rmd]: Ah, sorry, you'll need this `devtools::install_github("ropensci/plotly#296")`

**Comment by cpsievert** [comment]
> can we still keep an option for printing the object as an iframe with plotly_POST?

I added that in https://github.com/ropensci/plotly/commit/350b582b40d010815a67cd10868be26cab375359 (that pull request - https://github.com/ropensci/plotly/pull/296 - is ready to merge, BTW)

File: _posts/r/2015-07-30-embedding-in-html.Rmd (hunk 1 above)
Comment (chriddyp) [inline _posts/r/2015-07-30-embedding-in-html.Rmd]: got it, great, thanks 
image

**Comment by cpsievert** [comment]
Actually, thanks to [this post](http://brendanrocks.com/htmlwidgets-knitr-jekyll/), it might not be too bad to make htmlwidgets play nicely with Jekyll

File: _posts/r/2015-07-30-user-guide.Rmd [hunk 2]
```diff
@@ -9,65 +9,96 @@ page_type: example_index
...
+add_trace(p, y = fitted(m), name = "loess")
+```
+
+__plotly__ was designed with a [pure, predictable, and pipeable interface](https://dl.dropboxusercontent.com/u/41902/pipe-dsls.pdf) in mind, so you can also use the `%>%` operator to modify your plots:
+
+```{r}
+economics %>%
+  plot_ly(x = date, y = unemploy / pop) %>% 
+  add_trace(y = fitted(m)) %>%
+  layout(showlegend = F)
```
Comment (chriddyp) [inline _posts/r/2015-07-30-user-guide.Rmd]: nvm, wasn't working off of the branch.

**Comment by chriddyp** [comment]
sweet. I merged this into https://github.com/plotly/documentation/pull/215, so I'm going to close this and merge the other one.

**Pull request closed without merging**